trello.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pprint
+import requests
+
+pp = pprint.PrettyPrinter(indent=4)
+app_board_name='To Do App'
+trello_member='brendanmcdaniel'
```

Comment:
If it's user specific it would be nicer as an environment variable. But actually in this case, you should just replace this with `'me'`. Trello can work out who you are based on the auth token.